components/table/README.md

```diff
@@ -78,7 +78,7 @@ const example2Body = (
 ```
 
 ### References:
-- https://govuk-elements.herokuapp.com/alpha-beta-banners/
+- https://govuk-elements.herokuapp.com/data/
```

Comment:
should eventually be https://design-system.service.gov.uk/components/table/ but good to have it corrected for now 👍